locales/ar/common.json

```diff
@@ -76,7 +76,8 @@
             "start-playing": "ابدأ الاستماع",
             "stop-repeating": "توقف عن التكرار",
             "times": "مرات",
-            "verses-range": "مجموعة الآيات"
+            "verses-range": "مجموعة الآيات",
+            "volume-ctrl": "ضوابط الصوت"
```

Comment:
Thanks for adding all the locales, but only English translations need to be added. The rest of the locales, we will take of.